vgen/src/main/java/io/gapi/vgen/java/JavaContextCommon.java

```diff
@@ -218,7 +236,9 @@ public Builder setParams(
                 s_newVariable(
                     field.getType(),
                     "formatted" + LanguageUtil.lowerUnderscoreToUpperCamel(field.getSimpleName()),
-                    languageProvider.getDescription(field)));
+                    languageProvider.getDescription(field),
+                    languageProvider.getCollectionConfig(
+                        service, fieldNamePatterns.get(field.getSimpleName()))));
```

Comment:
To retrieve the InterfaceConfig - see here:
https://github.com/googleapis/toolkit/blob/master/vgen/src/main/java/io/gapi/vgen/java/JavaGapicContext.java#L341

The languageProvider.getCollectionConfig(service, name) method is calling languageProvider.getApiConfig().getInterfaceConfig(service).getCollectionConfig(name)